components/Markdown.tsx

```diff
@@ -14,7 +17,10 @@ export default async function Markdown(props: { children: string }) {
 
   return (
     <div
-      className="prose font-inter text-xs leading-relaxed dark:prose-invert md:text-sm xl:text-left"
+      className={clsx(
+        `prose font-inter leading-relaxed dark:prose-invert xl:text-left`,
+        props.className ? props.className : "text-sm sm:text-base",
+      )}
       dangerouslySetInnerHTML={{ __html: result.toString() }}
     />
```

Comment:
I'd suggest to do something like the one shown, so that when someone tries to override the bg-color let's say, it'd cause a weird behaviour the default text size classes are no longer applied and would cause confusion. Doing the below method would eliminate that.

```jsx
<div className="default classes...">
  <div 
    classsName={props.className}
    dangerouslySetInnerHTML={{ __html: result.toString() }}
  />
</div>
```

Reply:
Yeah that sounds great